run/src/adapt.ts

```diff
@@ -1,15 +1,27 @@
 import {Stream} from 'xstream';
 
+declare var window: any;
+
+function getGlobal(): any {
+  if(typeof window !== 'undefined') {
+    return window;
+  }
+  if(typeof global !== 'undefined') {
+    return global;
+  }
+  return this;
+}
+
 export interface AdaptStream {
   (s: Stream<any>): any;
 }
 
-let adaptStream: AdaptStream = x => x;
+getGlobal().Cyclejs.adaptStream = (x => x) as AdaptStream;
```

Comment:
`getGlobal().Cyclejs` could be undefined in which case we need to check and create it. I'd recommend actually putting this `Cyclejs` container inside the `getGlobal()` function, doing the conditional create there. 

Check e.g. 
https://github.com/cyclejs/cyclejs/blob/8deedf9c91f187f77b2506a0e859e472526b7056/run/src/index.ts#L223-L224